components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -204,6 +204,14 @@
  */
 @property(nonatomic, assign) BOOL shouldIncludeSafeAreaInInitialDrawerHeight;
 
+/**
+ A flag allowing clients to opt-in to generating a "sticky" view that is presented behind the status
+ bar, to prevent bottom drawer content from conflicting with the status bar content when an explicit
+ header has not been provided by the client (see Github issue #6434, or internal issue b/123186711,
+ for more context).
+ */
+@property(nonatomic, assign) BOOL shouldUseStickyStatusBar;
```

Comment:
Good point. If the client has provided a `headerViewController`, setting `shouldUseStickyStatusBar` to YES is a no-op and the behavior should work as if the client had only provided the `headerViewController` and not modified the flag. 

I think it may be useful to make more explicit (given that some clients, like us, may have a default generator for the bottom drawer that takes in a nullable `headerViewController`) and I have modified your suggested comment block to make this even more explicit, but let me know if you think the addition is unnecessary. 😅